### PR TITLE
chore: Fix and enhance tax and invoice information in wizard step 1

### LINKED ISSUE #7690: Schedule Page: User not able to schedule session when Microlocation is hidden
User not able to schedule session when Microlocation is hidden

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes https://github.com/fossasia/open-event-frontend/issues/7957

Follow up to https://github.com/fossasia/open-event-frontend/pull/7960

Screenshot from 2021-10-20 10-30-50
Screenshot from 2021-10-20 10-30-53
## part 3
Screenshot from 2021-10-20 10-31-06

#### Final
image

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Thank you!

File: app/mixins/event-wizard.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
 
       // model.tax is set in basic-detail step to workaround issue #4385
       let tax = this.model.tax || data.tax;
-      if (tax) {
+      if (tax && tax.name) {
```
[the remark this answers is not in the document]
Comment (maze-runnar) [reply app/mixins/event-wizard.js]: @pc-beast: previously it was added because, the tax were created at the time of event create (with empty names). so later if you go to create tax info, and it doesn't found the name, the modal will pop up as it will know that tax need to be created (edited in a more technical sense). But in tax refactor I have changed that flow because it was causing bugs. But kept this part, because it might cause issue for few edge cases that are off the eyes now. FYI @mariobehling

**Comment by mariobehling** [comment]
Ok, thank you. I am again merging this and we can test this further on the test deployment.
